crates/utils/src/tracing/grpc.rs

```diff
@@ -8,8 +10,22 @@ use crate::tracing::OpenTelemetrySpanExt;
 /// Additionally also pulls in remote tracing context which allows the server trace to be connected
 /// to the client's origin trace.
 pub fn grpc_trace_fn<T>(request: &http::Request<T>) -> tracing::Span {
-    // Create a span with a generic, static name.
-    let span = tracing::info_span!("rpc", rpc.uri = %request.uri());
+    // Create a span with a generic, static name. Fields to be recorded after needs to be
+    // initialized as empty since otherwise the assignment will have no effect.
+    let span = tracing::info_span!("rpc", rpc.uri = %request.uri(), otel.name = field::Empty);
```

Comment:
Let's also preset the `rpc.service` and `rpc.method` fields like this? aka 

`span.record("rpc.service, service);`

Reply:
Feel free to revert my commit that replaced that logic with the simple uri. I used it to debug on devnet, not required anymore